src/project/internal/projectconfiguration.cpp

```diff
@@ -63,6 +63,7 @@ static const Settings::Key GENERATE_AUDIO_TIME_PERIOD_TYPE_KEY(module_name, "pro
 static const Settings::Key NUMBER_OF_SAVES_TO_GENERATE_AUDIO_KEY(module_name, "project/numberOfSavesToGenerateAudio");
 static const Settings::Key SHOW_CLOUD_IS_NOT_AVAILABLE_WARNING(module_name, "project/showCloudIsNotAvailableWarning");
 static const Settings::Key DISABLE_VERSION_CHECKING(module_name, "project/disableVersionChecking");
+static const Settings::Key DISABLE_BACKUP(module_name, "project/disableBackup");
```

Comment:
I would prefer not using "negative" names for settings; this often leads to double negations which can be confusing and ambiguous. 

So it would be better to have a setting called `CREATE_BACKUP_BEFORE_SAVING(module_name, "project/createBackupBeforeSaving)`, with default value `true`.

(Incidentally, that would also match the MuseScore 3 functionality, but that's not the most relevant here, as far as I'm concerned.)

(And I understand that you took `DISABLE_VERSION_CHECKING` as an example; someday we should perhaps give that the same treatment of making it "positive", but better not now.)